wg-resource-management/README.md

```diff
@@ -0,0 +1,54 @@
+# WG Resource Management
+
+A working group that is a cross-SIG effort currently sponsored by sig-node and sig-scheduling with
+a focus on improving Kubernetes Compute Resource Management.
+
+Features tackled in this workgroup typically span more than one of the following SIGs:
+* SIG-Node
+* SIG-Scheduling
+* SIG-Autoscaling
+* SIG-Instrumentation
+* SIG-Storage
+* SIG-Networking
+
+Many advanced resource management topics span these functional areas, and this working
+group is intended to support collaboration. Prior to pursuing major changes in compute resource
+management, it is encouraged that community members share their ideas in this working group
+to reach broad consensus across the SIGs. Once consensus is reached, the enhancements
+are then managed by the appropriate SIGs based on impacted areas of code.
+
+It is an open and active working group, and we always welcome new additions!
+
+## Goals:
+
+Topics include, but are not limited to:
+
+* Support for Performance sensitive workloads (exclusive cores, cpu pinning strategies, numa)
+* Integrating new hardware devices (gpus, fpga, infiniband, etc.)
+* Improving resource isolation (local storage, hugepages, caches, etc.)
+* Improving Quality of Service (Performance SLOs)
+* Performance benchmarking
+* APIs and extensions related to the features mentioned above 
+* ...
+
+## Contact us:
+
+At this time, the group discussion primarily occurs on sig-node slack channel.
+
+* via [Slack](https://kubernetes.slack.com/messages/sig-node/)
+* via [Google Groups](https://groups.google.com/forum/#!forum/kubernetes-sig-node)
```

Comment:
+1 for a separate ML. 
As of now I specifically send all emails related to RM to multiple SIG groups. 
It will lead to more MLs, but on the other hand, a new ML for this workgroup will be very focused and can help reduce spam for folks not working on RM.